src/Water/latent_heat.jl

```diff
@@ -0,0 +1,45 @@
+const SLH_A0 = 2.5008e6    # J kg⁻¹     | Used for specific latent heat
+const SLH_A1 = -2.36e3     # J kg⁻¹ K⁻¹ | Used for specific latent heat
+const SLH_A2 = 1.6         # J kg⁻¹ K⁻² | Used for specific latent heat
+const SLH_A3 = -0.06       # J kg⁻¹ K⁻³ | Used for specific latent heat
+const SLH_I0 = 2.8341e6    # J kg⁻¹     | Used for specific latent heat
+const SLH_I1 = -290.0      # J kg⁻¹ K⁻¹ | Used for specific latent heat
+const SLH_I2 = -4.0        # J kg⁻¹ K⁻² | Used for specific latent heat
+
+
+
+
+"""
+    LH_v_ice(tem::FT)
```

Comment:
Some latent heats are defined in Common/Thermodynamics: 
https://github.com/CliMA/ClimateMachine.jl/blob/766ad0c0354742b08baab5a9d8aef10a511d86c6/src/Common/Thermodynamics/relations.jl#L13-L14

Could they be also used here?